src/Uno.UI/UI/Xaml/Input/Internal/FocusProperties.cs

```diff
@@ -292,7 +292,7 @@ internal static bool HasFocusedElement(DependencyObject element)
 		{
 			if (parent is UIElement uiElement)
 			{
-				return uiElement.GetChildrenInTabFocusOrderInternal();
+				return uiElement.GetChildrenInTabFocusOrderInternal() ?? Array.Empty<DependencyObject?>();
```

Comment:
This virtual method return type is annotated as non-null. The reason for the null as far as I can tell is one of the overrides which exists in a file with nullable-reference-types disabled.

I think it makes more sense if this line is left untouched, and instead update the following line:

https://github.com/unoplatform/uno/blob/e0b80489d13ea5d07f0fa5b22646a7946144e688/src/Uno.UI/Microsoft/UI/Xaml/Controls/Repeater/ItemsRepeater.cs#L896

and potentially add `#nullable enable` in it.

Reply:
I would suggest to check what WinUI is doing and handle the null in the same place.

Reply:
`ItemsRepeater` does return `null` here, so actually the the return type of `GetChildrenIntabFocusOrderInternal` should be nullable. I will enable nullable references and adjust the return type accordingly